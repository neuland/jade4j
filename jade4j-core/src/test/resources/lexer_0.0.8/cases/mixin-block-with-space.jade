mixin m(id)
  div
    block    

+m()
  | This text should appear