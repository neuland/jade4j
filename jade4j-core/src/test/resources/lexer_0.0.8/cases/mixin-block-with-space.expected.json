{"type":"mixin","line":1,"val":"m","args":"id"}
{"type":"indent","line":2,"val":2}
{"type":"tag","line":2,"val":"div","selfClosing":false}
{"type":"indent","line":3,"val":4}
{"type":"mixin-block","line":3}
{"type":"outdent","line":5}
{"type":"outdent","line":5}
{"type":"call","line":5,"val":"m","args":""}
{"type":"indent","line":6,"val":2}
{"type":"text","line":6,"val":"This text should appear"}
{"type":"outdent","line":6}
{"type":"eos","line":6}